server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/FetchHandler.scala

```diff
@@ -140,8 +143,9 @@ class FetchHandler(val conf: TransportConf) extends BaseMessageHandler with Logg
     val chunksBeingTransferred = streamManager.chunksBeingTransferred
     if (chunksBeingTransferred >= conf.maxChunksBeingTransferred) {
       logError(s"The number of chunks being transferred $chunksBeingTransferred" +
-        s"is above ${conf.maxChunksBeingTransferred()}.")
+        s"is above ${conf.maxChunksBeingTransferred}.")
       workerSource.stopTimer(WorkerSource.FetchChunkTime, req.toString)
+      // TODO will it cause a hang?
```

Comment:
cc @waitinfuture

Reply:
Yes, seems it will cause a RPC timeout. I think we should reply a FetchFailure immediately.